src/azure-cli/azure/cli/command_modules/acr/policy.py

```diff
@@ -4,35 +4,42 @@
 # --------------------------------------------------------------------------------------------
 
 from azure.cli.core.commands import LongRunningOperation
-from ._utils import validate_managed_registry
+from ._utils import validate_premium_registry
 
 
-POLICIES_NOT_SUPPORTED = 'Policies are only supported for managed registries.'
+POLICIES_NOT_SUPPORTED = 'Policies are only supported for managed registries in Premium SKU.'
 
 
 def acr_config_content_trust_show(cmd,
-                                  client,
                                   registry_name,
                                   resource_group_name=None):
-    _, resource_group_name = validate_managed_registry(
+    registry, _ = validate_premium_registry(
         cmd, registry_name, resource_group_name, POLICIES_NOT_SUPPORTED)
-    return client.list_policies(resource_group_name, registry_name).trust_policy
+    policies = registry.policies
+    trust_policy = policies.trust_policy if policies else None
```

Comment:
Do you want to return None or empty json?

Reply:
Does it matter? It's the same UX. Is there a standard?

Reply:
Changed to `{}`